docs/user/faq/troubleshooting.md

```diff
@@ -2,106 +2,118 @@
+  <br /><br />
+You can also check out our guides on how to configure VPN access <a href="/user/guides/access/vpn">here</a>.
+</div>
+  </div>
+</details>
+
+<details>
+  <summary>Can I recover a lost admin password?</summary>
+  <div>
+    <div>
+    You can not outright recover your admin password but you can regain access to your DAppNode if you wrote down the recovery token provided during the inital setup. 
+    <br /><br />
+    More about the initial setup can be found <a href="/user/quick-start/first-steps">here</a>
+    <br /><br />
+</div>
+  </div>
+</details>
+
+<details>
+  <summary>How do I recover a lost admin password using my recovery token?</summary>
+  <div>
+    <div>
+    Visit the login page of your DAppNode:
+    <br /><br />
+    <p align="center">
     <img src="../../../img/login.png"/>
-</p>
-
-We should select the option "Forgot password?". It will appear a field where you can reset your password using the recovery token. In this example, we would use the token <code>4LMB9w3l50Yljwr6bIgQ</code>. You have to use your recovery token.
+    </p>
 
+You will then see the option 'Forgot password?' underneath the 'Login' button. Once you click that, a new input field will appear where you can input your recovery token. We will use <code>4LMB9w3l50Yljwr6bIgQ</code> in this example.
 <p align="center">
     <img src="../../../img/reset_password_with_recovery_token.png"/>
 </p>
-
-After confirming the recovery token, you can define an admin user in DAppNode and his password in the same way you did the first time you connect.
-
+If you have entered the correct recovery token, you'll then be prompted to enter a new admin password.
+<br /><br />
 <p align="center">
     <img src="../../../img/sign_in_image.png"/>
 </p>
-
-Remember using a password with the next properties:
-
-- Minimum 8 characters
-- At least one Cap letter
-- At least one number
-
-Once you filled the field and press the register button, dappnode will provide a recovery token.
-
+Once you've filled out those fields and pressed 'Register', your DAppNode will once again provide you a recovery token. <b>Do not forget to write down and save this one!</b>
 <p align="center">
     <img src="../../../img/recovery_token.png"/>
 </p>
 
-We write down and save that token and we accept we have copied our recovery token. We log in with our new credentials and we have access to our DAppNode.
-
-## How to recover your recovery token and recover admin access
+</div>
+  </div>
+</details>
 
-In case we have lost both the password and the recovery token, we need to access our DAppNode: SSH into the DAppNode host or plug a keyboard.
-
-Once we are in our DAppNodeMachine, we type the next command which shows us the recovery token:
+<details>
+  <summary>What do I do if I lost the password AND my token?!</summary>
+  <div>
+    <div>
+In the case where you have lost both the password and the recovery token, we will need either SSH or local access to the DAppNode machine.
```

Comment:
```suggestion
In the case where you have lost both the password and the recovery token, you will need either SSH or local access to the DAppNode machine.
```

Reply:
i think we should have all references be to you not we as we dont want to confused people thinking the team has access or control